tests/__snapshots__/test_help_messages.ambr

```diff
@@ -7105,79 +7105,6 @@
   ╰──────────────────────────────────────────────────────────────────────────────╯
   
   
-  '''
-# ---
-# name: test_help_messages[streamlit.'\x07']
```

Comment:
why is this deleted?

Reply:
Looks like it was recently added in #1238 -- seems like a mistake to me. It's also removed when running `pytest --snapshot-update` on `main`.
@sfc-gh-turbaszek Can you confirm?

Reply:
What's the latest on this?

Reply:
I can't replicate this on current main. Nevertheless I will change test so we are treating unused snapshots as error.

Reply:
Removed the snapshot update from this PR, but this help section is still removed for me on main after running `--snapshot-update`